java_training/jan20/Employee_mgmt_sql

```diff
@@ -0,0 +1,84 @@
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+CREATE SCHEMA IF NOT EXISTS `try1` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci ;
```

Comment:
Here is the MySQL naming conventions that Leapfrog Database services has created. Please follow these guides lines https://lftechnology.atlassian.net/wiki/display/DATA/Standard+Naming+Convention